witness/src/Dscp/Resource/Keys/Functions.hs

```diff
@@ -47,11 +48,8 @@ fromSecretJson pp KeyJson{..} = do
 -- | Where keyfile would lie.
 storePath
     :: (HasWitnessConfig, Buildable (Proxy node))
-    => BaseKeyParams -> Proxy node -> FilePath
-storePath BaseKeyParams{..} nodeNameP =
-    fromMaybe defPath bkpPath
-  where
-    defPath = giveLC @'["core", "generated", "home"] @WitnessConfig </> (nodeNameP |+ ".key")
+    => AppDir -> Proxy node -> FilePath
+storePath appDir nodeNameP = appDir </> (nodeNameP |+ ".key")
```

Comment:
@nasportivnoy, removing `fromMaybe` here broke `--faucet-keyfile` and similar options which allowed specifying custom secrets location.
I fixed that regression in #174.
Still, if you had some ulterior motives for that, some explanation would really help.